neuro_flow/storage.py

```diff
@@ -151,7 +163,7 @@ async def fetch_bake_by_id(self, project: str, bake_id: str) -> Bake:
         pass
 
     @abc.abstractmethod
-    async def fetch_config(self, bake: Bake) -> str:
+    async def fetch_configs(self, bake: Bake) -> List[Tuple[RemotePath, str]]:
```

Comment:
`List` -> `Sequence`
Let's forbid things like 
```
ret = await storage.fetch_configs(bake)
ret.append(additional_result)
```
I prefer an explicit list construction for such case

Reply:
OK, got it. 
Some justification for initial solution: I'm trying to follow the rule: accept the most generic type you can, return the most specific type that is safe to return. But I agree that returning list can be unsafe in case of some "InMemory" storage impl.
Thanks for making me to think about this!

Reply:
You are right, but sometimes we `Sequence` trick to declare a suggested-to-be-immutable return value without converting it to the real tuple.